recipe/meta.yaml

```
@@ -38,17 +41,20 @@ test:
     - if not exist %LIBRARY_BIN%\\FLAC++.dll exit 1   # [win]
     - if not exist %LIBRARY_LIB%\\FLAC.lib exit 1     # [win]
     - if not exist %LIBRARY_BIN%\\FLAC.dll exit 1     # [win]
-    - flac --version                                  # [build_platform == target_platform]
+    - flac --version
 
 about:
   home: https://www.xiph.org/
-  license: BSD-3-Clause
-  license_family: BSD
-  license_file: COPYING.Xiph
+  license: BSD-3-Clause AND GPL-2.0-or-later AND GFDL-1.1-or-later
+  license_family: OTHER
+  license_file: 
+    - COPYING.Xiph
+    - COPYING.FDL
+    - COPYING.GPL
```

```suggestion
  license: BSD-3-Clause AND GPL-2.0-or-later AND GFDL-1.1-or-later AND LGPL-2.1-or-later
  license_family: OTHER
  license_file: 
    - COPYING.Xiph
    - COPYING.FDL
    - COPYING.GPL
    - COPYING.LGPL
```
As highlighted in https://github.com/xiph/flac/blob/master/COPYING.LGPL and https://github.com/xiph/flac/blob/master/src/share/grabbag/cuesheet.c.